common.gypi

```diff
@@ -12,7 +12,7 @@
       # More details at https://github.com/mapbox/mason/issues/319
       '-D_GLIBCXX_USE_CXX11_ABI=0'
     ],
-    'cflags_cc!': ['-std=gnu++0x','-fno-rtti', '-fno-exceptions'],
+    'cflags_cc!': ['-std=gnu++0x', '-std=gnu++1y', '-fno-rtti', '-fno-exceptions'],
```

Comment:
What does it do when you specify two language standards at the same time?

Reply:
Oh oops this might be leftover from my attempts to get this working on travis with the latest xcode version. I'll remove and confirm it keeps working.

Reply:
removed in https://github.com/mapbox/vtcomposite/pull/113/commits/e4720334e9a2b1aab114302946c2ac6f44fdff4e